### ISSUE TITLE
Improve the layout of the cases (MainLayout)

### ISSUE THREAD

**Issue by jcgueriaud1** [parttio/accessibility-checker-for-vaadin#13]
The MainLayout throws some accessibility errors and should be improved

**Comment by jcgueriaud1** [comment]
![image](https://github.com/parttio/accessibility-checker-for-vaadin/assets/51313578/3cadb824-7a66-4382-b809-640283095228)
